src/hotspot/cpu/aarch64/nativeInst_aarch64.hpp

```diff
@@ -655,7 +655,7 @@ class NativeLdSt : public NativeInstruction {
       return 0;
     }
   }
-  size_t size_in_bytes() { return 1 << size(); }
+  size_t size_in_bytes() { return 1ull << size(); }
```

Comment:
Capital ULL - I find that easer to search for and it is more consistent.

Reply:
Thank you! Fixed.